internal/network/command_private_link.go

```diff
@@ -0,0 +1,20 @@
+package network
+
+import (
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+)
+
+func newPrivateLinkCommand(prerunner pcmd.PreRunner) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "private-link",
+		Short:   "Manage private links.",
+		Aliases: []string{"pl"},
+		Args:    cobra.NoArgs,
+	}
+
+	cmd.AddCommand(newPrivateLinkAccessCommand(prerunner))
```

Comment:
Are we adding more subcommands here in the future?

Reply:
Yes, we will be adding `confluent network private-link attachment` and `confluent network private-link attachment connection`.